docs/account-management/auth/connect-to-gdn.md

```diff
@@ -82,7 +94,7 @@ EMAIL = "nemo@nautilus.com"
 PASSWORD = "xxxxxx"
 AUTH_TOKEN = "bearer "
 
-# Create a HTTPS Session
+# Create an HTTPS Session
 url = "{}/_open/auth".format(FED_URL)
 payload = {
 'email':EMAIL,
```

Comment:
```suggestion
  'email':EMAIL,
```

Reply:
DAmn, I knew I should have waited. I'll put these changes in a new PR.

Reply:
same needs to be applied for password (the below line) and 'content-type' (line no. 104)